terraform/modules/iam_user/read_only_admin/user.tf

```diff
@@ -0,0 +1,169 @@
+                "elasticmapreduce:ListInstances",
+                "glacier:List*",
+                "glacier:DescribeVault",
+                "glacier:GetVaultNotifications",
+                "glacier:DescribeJob",
+                "glacier:GetJobOutput",
+                "iam:Get*",
+                "iam:List*",
+                "iam:GenerateCredentialReport",
+                "kinesis:ListStreams",
+                "kinesis:DescribeStream",
+                "kinesis:GetShardIterator",
+                "kinesis:GetRecords",
+                "kms:Get*",
+                "kms:List*",
+                "lambda:ListFunctions",
+                "rds:Describe*",
+                "rds:ListTagsForResource",
+                "redshift:Describe*",
+                "redshift:ViewQueriesInConsole",
+                "route53:ListHealthChecks",
+                "route53:ListHostedZones",
+                "route53:ListResourceRecordSets",
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:DeleteObject",
+                "s3:GetBucketACL",
+                "s3:GetBucketLocation",
+                "s3:GetBucketLogging",
+                "s3:GetBucketPolicy",
+                "s3:GetBucketTagging",
+                "s3:GetBucketWebsite",
+                "s3:GetBucketNotification",
+                "s3:GetLifecycleConfiguration",
+                "s3:GetNotificationConfiguration",
+                "s3:GetObject",
+                "s3:GetObjectMetadata",
+                "s3:List*",
+                "sdb:ListDomains",
+                "sdb:DomainMetadata",
+                "ses:ListIdentities",
+                "ses:GetSendStatistics",
+                "ses:GetIdentityDkimAttributes",
+                "ses:GetIdentityVerificationAttributes",
+                "ses:GetSendQuota",
+                "sns:GetSnsTopic",
+                "sns:GetTopicAttributes",
+                "sns:GetSubscriptionAttributes",
+                "sns:ListTopics",
+                "sns:ListSubscriptionsByTopic",
+                "sqs:ListQueues",
+                "sqs:GetQueueAttributes",
+                "storagegateway:Describe*",
+                "storagegateway:List*",
+                "support:*",
+                "swf:ListClosedWorkflowExecutions",
+                "swf:ListDomains",
+                "swf:ListActivityTypes",
+                "swf:ListWorkflowTypes",
+                "workspaces:DescribeWorkspaceDirectories",
+                "workspaces:DescribeWorkspaceBundles",
+                "workspaces:DescribeWorkspaces"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+        },
+        {
+            "Sid": "CloudWatchLogsSpecific",
+            "Effect": "Allow",
+            "Action": [
+                "logs:GetLogEvents",
+                "logs:DescribeLogGroups",
+                "logs:DescribeLogStreams"
+            ],
+            "Resource": [
+                "arn:aws:logs:*:*:*"
```

Comment:
We shouldn't hard-code partitions like `aws`, since this would be `aws-us-gov` in govcloud. We've been using the `aws_partition` variable to handle this elsewhere. Also, just curious--why does cloudwatch go in its own statement?